sdk/translation/azure-ai-translation-text/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "java",
   "TagPrefix": "java/translation/azure-ai-translation-text",
-  "Tag": "java/translation/azure-ai-translation-text_a2b0c1776a"
+  "Tag": "java/translation/azure-ai-translation-text_177e917101"
```

Comment:
Please add a tracking GitHub issue to re-record and update the azure-core-test dependency to use the new test proxy.